units/patch_intent.go

```diff
@@ -493,7 +493,10 @@ func (j *patchIntentProcessor) processTriggerAliases(ctx context.Context, p *pat
 	}
 
 	for _, intent := range triggerIntents {
-		if err := j.env.RemoteQueue().Put(ctx, NewPatchIntentProcessor(mgobson.ObjectIdHex(intent.ID()), intent)); err != nil {
+		job := NewPatchIntentProcessor(mgobson.ObjectIdHex(intent.ID()), intent)
```

Comment:
I was thinking we could do this _only_ in the case of `parentStatus == ""`
That way we could have the performance advantages of separating the processing at least in the case where the child patch is anyways waiting for the parent patch to complete.

Reply:
ah yes, I did that, but looks like I didn't save my file before committing. I'll fix. Thanks for pointing it out!